bootstrap/eks/config/rbac/serviceaccount.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: controller-manager
+  namespace: system
+  labels:
+    control-plane: controller-manager
+  annotations:
+    ${AWS_CONTROLLER_IAM_ROLE/#arn/eks.amazonaws.com/role-arn: arn}
```

Comment:
Can we add a comment to this one, as this is a very special hack that is specific to https://github.com/drone/envsubst, and doesn't work with gettext?

Reply:
Added. 

Is there any scenario where this would be used with envsubst from gettext? Or are you just thinking about potential issues in the future?

Reply:
potential issues in the future. when we started using envsubst in clusterctl and  that was within a v1alpha3 release, so in case something happened again it'd be a little easier to diagnose.

Reply:
Good to know! 👍